File: QE-SELENIUM-JAVA/demo/src/main/java/demos/Activity1.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {

    public static void main(String[] args) {
        FirefoxDriver driver = new FirefoxDriver();    
        driver.get("https://training-support.net");
        System.out.println("Home Page Title: " + driver.getTitle());
        WebElement aboutLink = driver.findElement(By.linkText("About Us"));
        aboutLink.click();
        System.out.println("About Page Title: " + driver.getTitle());
        driver.quit();
    }
}

File: QE-SELENIUM-PYTHON/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By
driver = webdriver.Firefox()
driver.get("https://training-support.net")
print("Home Page Title:", driver.title)
about_link = driver.find_element(By.LINK_TEXT, "About Us")
about_link.click()
print("About Page Title:", driver.title)

driver.quit()